### ISSUE TITLE
No disk-cache of files

### ISSUE THREAD

**Issue by stefansolid** [tonyfederer/OutlookFileDrag#30]
Is it possible to adjust the code to cache files in memory? We'd like to use this plugin however due to some regulations we have on governmental level had troubles with caching files on disk is not accepted. Therefore we're unable to use this plugin as is. Is this technically feasible to do?

**Comment by tonyfederer** [comment]
Unfortunately browsers only support drag and drop of physical files (CF_HDROP format).  Outlook natively uses virtual files (CFSTR_FILEDESCRIPTORW format) since the Outlook items do not exist on disk.

This plugin detects a drag and drop has started, caches the file to disk, and then adds the physical file to the available formats that can be dropped.

If browsers supported the CFSTR_FILEDESCRIPTORW format, then this plugin would not be necessary and the files would never be written to disk.

I did a quick search and was surprised to find that Edge actually natively supports drag and drop from Outlook items as virtual files.  I can't find when they added this feature, but I confirmed I was able to drag and drop from Outlook to OneDrive in the Edge browser without OutlookFileDrag.

Now that Microsoft is switching to using the open-source Chromium project as the rendering engine for Edge, they are contributing this functionality to the that project as well.  This means the feature should be coming to Chrome, soon, as well.

[Chromium Commit 1487798: Drag and Drop support for emails and attachments from Microsoft Outlook](https://chromium-review.googlesource.com/c/chromium/src/+/1487798)
